src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1603,7 +1559,7 @@ public void RegisterAllPorts()
         /// <param name="data"></param>
         /// <param name="index"></param>
         /// <returns></returns>
-        public PortModel AddPort(PortType portType, PortData data, int index)
+        private PortModel AddPort(PortType portType, PortData data, int index)
```

Comment:
API Change: `NodeModel.AddPort(...)` has been removed. Ports can now be added using `NodeModel.InPorts.Add(...)`